Topbar

scosy opened the pull request:
Removed title from the topbar so it is responsive.

Commit:
topbar is responsive

Commit:
conflict resolved

Commit:
added a div around the image so it is well placed

Hannah64 reviewed (changes requested):
Display the title in laptop and tablet format only
Display smaller logo in phone format
Remove the frame and add background color
Keep good size for input in any format

Commit:
Merge branch 'dev' of github.com:wcs-projet-2/wcs-project-2 into topbar

Commit:
removed title when passing in mobile mode
checkboxes only appear in column when in mobile mode
input keeps a good size even when in mobile

Commit:
topbar is now responsive

Commit:
Merge branch 'dev' of github.com:wcs-projet-2/wcs-project-2 into topbar

Commit:
resolved issue with large screens

Commit:
Merge branch 'dev' of github.com:wcs-projet-2/wcs-project-2 into topbar

Commit:
resolved conflict

Commit:
done

Hannah64 reviewed (approved):
Good job

Malou64 reviewed (approved):
ok
